tox.ini

```diff
@@ -12,6 +12,8 @@ deps =
     -Urrequirements_dev.txt
     # Requests is required for the util script
     requests
+    # Add the example plugin
+    -e examples/sqlfluff-plugin-example

```

Comment:
Installed it as editable so people can change the example plugin and not worry about having to re-install before running tox